docs/blocks/data_loading.md

```diff
@@ -0,0 +1,993 @@
+
+This function will load at maximum 100,000 rows of data (this limit is configurable). To operate on more data, consider performing data transformations in warehouse using `execute`.
+
+-   **Args**:
+    -   `query_string (str)`: Query to fetch a table or subset of a table.
+    -   `limit (int, Optional)`: The number of rows to limit the loaded data frame to. Defaults to 100,000.
+    -   `*args, **kwargs`: Additional parameters to pass to the query. See [Snowflake Connector Docs](https://docs.snowflake.com/en/user-guide/python-connector-api.html#execute) for additional parameters.
+-   **Returns**: (`DataFrame`) Data frame containing the queried data
+
+**_open_** - `open()`
+
+Opens a connection to Snowflake servers.
+
+**_rollback_** - `rollback()`
+
+Rolls back (deletes) all database changes made since the last transaction.
+
+**_sample_** - `sample(schema: str, table: str, size: int, **kwargs) -> DataFrame`
+
+Sample data from a table in the Snowflake warehouse. Sample is not guaranteed to be random.
+
+-   **Args**:
+
+    -   `schema (str)`: The schema to select the table from.
+    -   `size (int)`: The number of rows to sample. Defaults to 100,000
+    -   `table (str)`: The table to sample from in the connected database.
+
+-   **Returns**: (`DataFrame`) Sampled data from the data frame.
+
+# Configuration Settings
+
+Connections to third-party data storage require you to specify confidential information such as login information or access keys. While you can manually specify this information code while constructing data loading clients, it is recommended to not store the secrets directly in code.
+
+Instead, Mage provides **configuration loaders** which allow data loading clients to use your secrets without explicitly writing them in code.
+
+Currently, the following sources (and their corresponding configuration loader) can be used to load configuration settings:
+
+-   Configuration File - `ConfigFileLoader`
+-   Environment Variables - `EnvironmentVariableLoader`
+-   AWS Secrets Manager - `AWSSecretLoader`
+
+For example, the code below constructs a Redshift data loading client using secrets stored in AWS Secrets Manager
+
+```python
+from mage_ai.io.config import AWSSecretLoader
+from mage_ai.io.redshift import Redshift
+
+config = AWSSecretLoader()
+loader = Redshift.from_config(config)
+```
+
+The following are the set of allowed key names that you must name your secrets with in order for Mage's configuration loaders to recognize your secrets. In code you can refer to these keys by their string name or using the `mage_ai.io.config.ConfigKey` enum. Not all keys need be specified at once - only use the keys related to the services you utilize.
+
+| Key Name                        | Service                      | Client Constructor Parameter | Description                                                             | Notes                                       |
```

Comment:
The table is malformed 
<img width="1060" alt="image" src="https://user-images.githubusercontent.com/80284865/180889636-aa8ee707-60ca-4270-95cf-3876f6b562a2.png">